runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -203,29 +210,33 @@ void __catalyst__rt__finalize()
 
 void __catalyst__rt__device_init(int8_t *rtd_lib, int8_t *rtd_name, int8_t *rtd_kwargs)
 {
+    using namespace Catalyst::Runtime;
```

Comment:
Should we move this top-level to avoid repetition? (since this is a .cpp file it won't affect other translation units)

Reply:
Done!